std/range/package.d

```diff
@@ -6629,10 +6627,10 @@ if (isForwardRange!RangeOfRanges &&
         }
     }
 
-    // ElementType opIndex(size_t n)
-    // {
-    //     return _input[n].front;
-    // }
+    static if (isRandomAccessRange!RangeOfRanges)
+    {
+        auto ref opIndex(size_t n) { return _input[n]; }
```

Comment:
I don't think this is right at all. For instance, I would expect this:

```D
auto x = [[1,2,3],[4,5,6]].transposed;
assert(x.front.equal([1,4])); // passes today
assert(x[0].equal([1,4])); // I don't think this would pass with your change.
```